templates/login.html

```diff
@@ -3,21 +3,86 @@
 <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/all.css" integrity="sha384-lKuwvrZot6UHsBSfcMvOkWwlCMgc0TaWr+30HWe3a4ltaBwTZhyTEggF5tJv8tbt" crossorigin="anonymous">
 {% endblock %}
 
+{% block scripts%}
+<script type="text/javascript"> 
+    
+    
+
+
+  
+
+    function search(){
+
+        console.log("Hello")
+        var url= "/api/search"
+        var searchform = new FormData( document.getElementById("searchform") );
+        console.log(searchform)
+        var usersearch = searchform.get("book");
+        console.log(usersearch)
+        var responseobj = {}
+        responseobj['query'] = usersearch;
+        var json = JSON.stringify(responseobj);
+   
+        console.log(json)
```

Comment:
remove all the console.log statements as users might get access to some vital information. It's an unsafe practice and should be removed after testing the code.